config/locales/devise/en.yml

```diff
@@ -48,14 +46,13 @@ en:
     registrations:
       close_window: You can close this window if you are done.
       destroy_confirm: Deleting your account cannot be undone. All data associated
-        with your account will be removed. Are you sure you’d like to delete your
+        with your account will be removed. Are you sure you want to delete your
         account?
       destroyed: Your account has been successfully deleted.
       email_and_phone_need_confirmation: Before we finish updating your account, we
-        need to confirm both your new number and new email. Please follow the instructions
-        below to confirm your new number, then check for an email from us. Follow
-        the link in the email to confirm your new email address.
-      email_update_needs_confirmation: You updated your account, but we need to confirm
+        need to confirm both your new phone number and your new email address. Please follow the instructions
+        below to confirm your phone number. To confirm your new email address, check for an email from us and follow the link provided.
+      email_update_needs_confirmation: You have updated your account, but we need to confirm
```

Comment:
It worries me that this line is green. There's a rake task to check translations that may be helpful to run. Unfortunately, I can't recall what it is.